src/ng/compile.js

```diff
@@ -1386,6 +1386,75 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return TTL;
   };
 
+  var COMMENT_DIRECTIVES_ENABLED = true;
```

Comment:
Note to self: Using uppercase for non-constant values --> Not intuitive :smile:
(Can be addressed in a separate PR.)